app/models/chat_channel.rb

```diff
@@ -2,7 +2,7 @@ class ChatChannel < ApplicationRecord
   include AlgoliaSearch
   attr_accessor :current_user, :usernames_string
 
-  has_many :messages
+  has_many :messages, dependent: :destroy
```

Comment:
This is the most important change. 

Is there a reason why we delete instead of destroy messages?

Let me know if I should revert this.

Reply:
Is this comment for the next change? It seems like in this case there was no deleting or destroying.

Reply:
Yeah, thanks for the catch, the right question is: is there a way why we keep the messages instead of deleting them when a channel is deleted?